.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions: # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+
+    steps:
+      # Seed the build number with last number from TeamCity.
+      # This env var is used by the JS, and SBT builds, and guardian/actions-riff-raff.
+      # Set the value early, rather than `buildNumberOffset` in guardian/actions-riff-raff, to ensure each usage has the same number.
+      # For some reason, it's not possible to mutate GITHUB_RUN_NUMBER, so set BUILD_NUMBER instead.
+      - name: Set BUILD_NUMBER environment variable
+        run: |
+          LAST_TEAMCITY_BUILD=145
+          echo "BUILD_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "adopt"
+          java-version: "8"
+          cache: "sbt"
+
+      - name: AWS Auth
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+
+      - name: Build and Test project
+        run: |
+          sbt clean compile test universal:packageBin
```

Comment:
You may want to use [semicolons](https://www.scala-sbt.org/1.x/docs/Howto-Running-Commands.html#Provide+multiple+commands+to+run+consecutively) here to ensure the commands execute sequentially, and/or use [batch mode](https://www.scala-sbt.org/1.x/docs/offline/Running.html#Batch+mode) to ensure all the commands are treated as a single command - example [here](https://github.com/guardian/editorial-preferences/blob/main/.github/workflows/ci.yml#L35).

Reply:
Interesting - there's also a lot of examples of this working without the semicolons, and looking at the logs it seems to run in the correct sequence, as well as already [use batch mode ](https://github.com/guardian/flexible-snapshotter/actions/runs/6341675084/job/17225837821?pr=37#:~:text=Executing%20in%20batch%20mode.%20For%20better%20performance%20use%20sbt%27s%20shell)by default I guess?

Reply:
Yes it seems semicolons and batch mode are unnecessary in this case - see discussion [here](https://github.com/guardian/grid/pull/4146#discussion_r1344340943) if you're interested!

However it also looks like colon syntax is [deprecated](https://github.com/guardian/grid/pull/4146#discussion_r1343979088), and slash syntax should be used instead - more [here](https://github.com/scalameta/sbt-scalafmt/issues/137). Which I think should apply because the sbt version for this project is 1.6.2.

Reply:
Thanks for the links! So looking at the grid example, if I'm understanding correctly this would be ok?

`sbt clean compile test Universal/packageBin`